CONTRIBUTING.md

```diff
@@ -15,9 +15,9 @@
 
 ## WET v4.0
 
-* [Créer un &#171;&#160;pull request&#160;&#187;](http://wet-boew.github.io/wet-boew/docs/pull-fr.html)
+* [Créer un &#171;&#160;pull request&#160;&#187;](https://wet-boew.github.io/wet-boew/docs/pull-fr.html)
 
 ## WET v3.1
 
 * [Développer pour la BOEW](http://wet-boew.github.io/v3.1-ci/docs/gs-cd/dev-fr.html)
```

Comment:
What about this one? May as well do all the links in the document at the same time